### ISSUE TITLE
index.es.js:1033 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'width')

### ISSUE THREAD

**Issue by web-shoharab-pk** [diegomura/react-pdf#2434]
**Is your feature request related to a problem? Please describe.**
When I tried to print it using the print-JS package I got this error for this image tag. 
Note: I used same Image tag in another section but I didn't get any error

**Describe the solution you'd like**
  <View style={styles.signature__containt__data}>
         <Image src={src} style={styles.bar_code_image} />
         </View>

**Describe alternatives you've considered**

**Additional context**
![image](https://github.com/diegomura/react-pdf/assets/76746357/223341bd-cb19-4c06-bb2a-68f53462a174)

**Comment by TheBrainChain** [comment]
Same error with the following block of code:
```
<View
  render={({ pageNumber }) => (
    <View>
      <Image src={ReportHeader} />
    </View>
   )}
/>
```
No issue with:

```
<View
  render={({ pageNumber }) => (
    <View>
      <Text>{pageNumber}</Text>
    </View>
  )}
/>
```

or

```
<View>
  <Image src={ReportHeader} />
</View>
```

**Comment by vilenleung** [comment]
i had fact same issue.

**Comment by jrtell1** [comment]
Yep I ran also into this issue, wrote a more detailed bug report with REPL link: https://github.com/diegomura/react-pdf/issues/2444

This should probably not be tagged "new feature"
